src/panels/config/devices/ha-config-device-page.ts

```diff
@@ -80,6 +80,8 @@ export interface EntityRegistryStateEntry extends EntityRegistryEntry {
 
 export interface DeviceAction {
   href?: string;
+  target?: string;
+  rel?: string;
```

Comment:
Why is this an option when its hardcoded to "noreferrer" the only place that's defining it?